tests/lib/assertions/nested-amd64.model

```diff
@@ -6,16 +6,18 @@ model: pc
 architecture: amd64
 gadget: pc
 kernel: pc-kernel
-timestamp: 2019-05-16T18:06:04+00:00
-sign-key-sha3-384: 0u-IKZyVlTxwiBVOv7oT1FNKdfkKJyOSwbEWJPurlvtaGZxX6VrMHSESnIPeaNMC
+serial-authority:
+  - generic
```

Comment:
It's actually not strictly needed for this PR, I thought it was but it's not, the uc16 cloud-init test is failing for other reasons, but this is a good change to have in any case since it means that the devices get serial assertions which is closer to real devices anyways

Reply:
Thanks, I was a bit confused as I did a similar change such that we get serial numbers for remodel, and it wasn't clear how this change fits in this PR as none of code changes introduced here seem to require serials.